sdk/provisioning/Azure.Provisioning/tests/Infrastructure/WebSiteUsingL2/resources/rg_TEST/rg_TEST.bicep

```diff
@@ -1,10 +1,10 @@
 @secure()
 @description('SQL Server administrator password')
-param sqlAdminPassword string
+param sqlAdminPassword string = 'password'
 
 @secure()
 @description('Application user password')
-param appUserPassword string
+param appUserPassword string = 'password'
```

Comment:
this default value shouldn't be necessary especially for secure params

Reply:
I can pass the param in the validate call, but didn't want to have to special case per test.